apps/api/src/runs/run.entity.ts

```diff
@@ -59,6 +60,9 @@ export class RunEntity implements Run {
   @OneToMany(() => FileEntity, (file) => file.run)
   files: FileEntity[];
 
+  @OneToMany(() => DependencyProjectEntity, (dependentProject) => dependentProject.run)
+  dependencyProjects: DependencyProjectEntity[];
```

Comment:
there's no concept of projects currently, sounds a bit odd to add it just to manage task inter-dependencies.

Reply:
Will rename it to dependencyTasks